core/src/fr/hammons/slinc/modules/LinkageTools.scala

```diff
@@ -138,10 +140,12 @@ object LinkageTools:
       suffixCandidates: Seq[String],
       archMarks: Set[String]
   ): Seq[String] =
-    suffixCandidates.flatMap { suffix =>
-      if archMarks.isEmpty then Seq(s"$location$suffix")
-      else archMarks.map(archMark => s"${location}_$archMark$suffix")
-    }
+    val withoutArchMarks = suffixCandidates.map(sfx => s"$location$sfx")
```

Comment:
Please remove the "withoutArchMarks" resolution. The arch mark should always be necessary for this feature to help avoid a file being loaded for the wrong arch.

Reply:
done.